tests/e2e/utils/common_utils.go

```diff
@@ -25,5 +25,5 @@ func ReplaceSecretDataNewLineWithCarriageReturn(data []byte) []byte {
 
 // Extract tar.gz file to a directory of the same name in the same directory
 func ExtractTarGz(pathToDir, tarGzFileName string) error {
-	return exec.Command("tar", "-xzf", pathToDir + "/" + tarGzFileName, "-C", pathToDir).Run()
+	return exec.Command("tar", "-xzf", pathToDir+"/"+tarGzFileName, "-C", pathToDir).Run()
```

Comment:
was this from `go fmt`?

Reply:
Must have been -- maybe run indirectly by `make test` or `make bundle`

Reply:
`make test` runs `go fmt`